.github/master-label-triggers.yml

```diff
@@ -0,0 +1,313 @@
+# Add 'needs review' label to any pull request
+needs review:
+- "*"
+
+# Add 'repository' label to any root file changes
+repository:
+- ./*
+
+# Add 'github' label to any .github/ file changes
+github:
+- .github/*
+- .github/**/*
+
+# Add 'mkosi' label to any .mkosi/ file changes
```

Comment:
This looks like it would not apply to dracut. Is that from systemd?

Reply:
I'm planning on submitting mkosi setup pr request for dracut project to make it somewhat easer develop and test dracut on multiple distributions hence I ( prematurely ) including that line from having to do it later in the process.